_posts/2017-03-31-introducing-deckgl-v4.md

```diff
@@ -33,75 +33,74 @@ is provided and we expect the upgrade effort for applications to be minimal.
 
 ## A Growing Layer Catalog
 
-From the very beginning, deck.gl has always been a geospatial data visualization framework, and geospatial use cases will always be the top priority. The new layers added to the [layer catalog](https://uber.github.io/deck.gl/#/documentation/layer-catalog) are widely used in geospatial data analysis within Uber and we hope they will help solve the problems that deck.gl users face every day.
+From the very beginning, deck.gl has always been a geospatial data visualization framework, and geospatial use cases will always be our top priority. The new layers added to the [layer catalog](https://uber.github.io/deck.gl/#/documentation/layer-catalog) are widely used in geospatial data analysis within Uber and we hope they will help solve the problems that every deck.gl user faces every day.
 
 ### [`GeoJsonLayer`](https://uber.github.io/deck.gl/#/documentation/layer-catalog/geojson-layer)
 
-<p>
-	<img width="400" src="../img/demo-thumb-geojson.jpg">
+<p aligh="center">
+	<img width="900" src="../img/screenshot-geojson.png">
 </p>
 
-The GeoJsonLayer is designed to render geometry primitives in the widely accepted GeoJson data format. The GeoJsonLayer itself is very powerful as it renders multiple types of geometry primitives, including points, lines, multilines, polygons and multipolygons all at once. Moreover, it can render polygons and multipolygons in different styles, currently supporting filled and outline styles in 2D, and filled and wireframe styles in extruded.
+The GeoJsonLayer is designed to render geometry primitives in the widely accepted GeoJson data format. The GeoJsonLayer itself is very powerful as it renders multiple types of geometry primitives, including points, lines, multilines, polygons and multipolygons all at once. Moreover, it can render polygons and multipolygons in different styles, currently supporting filled and outline styles in 2D flat, and filled and wireframe styles in extruded.
```

Comment:
> currently supporting filled and outline styles in 2D flat, and filled and wireframe styles in extruded

Yes this needed fixing. "2D flat" is still an odd term. Maybe even clearer:

`currently supporting filled and outline styles in non-extruded (2D) mode, and filled and wireframe styles in extruded (3D) mode.`